litmus-portal/frontend/src/components/ChooseWorkflow/index.tsx

```diff
@@ -0,0 +1,305 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Typography, TextField, Modal } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+import Button from '@material-ui/core/Button';
+import { withStyles, Theme } from '@material-ui/core/styles';
+import { purple } from '@material-ui/core/colors';
+import useStyles from './styles';
+import useActions from '../../redux/actions';
+import * as WorkflowActions from '../../redux/actions/workflow';
+import PredifinedWorkflows from '../PredifinedWorkflows';
+import {
+  preDefinedWorkflowData,
+  workflowDetails,
+} from '../../models/predefinedWorkflow';
+// import { getWkfRunCount } from "../../utils";
+
+const CssTextField = withStyles({
```

Comment:
These methods could be added in styles and exported separately from there

Reply:
Done.